packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -240,8 +200,7 @@ class Actions extends InheritedWidget {
   /// The `context` argument must not be null.
   static ActionDispatcher of(BuildContext context, {bool nullOk = false}) {
     assert(context != null);
-    final InheritedElement inheritedElement = context.ancestorInheritedElementForWidgetOfExactType(Actions);
-    final Actions inherited = context.inheritFromElement(inheritedElement);
+    final Actions inherited = context.inheritFromWidgetOfExactType(Actions);
```

Comment:
Using inheritFromWidgetOfExactType because we context to rebuilt if Actions changes?

If that's true then Actions probably needs an operator== and hashCode overloads so the framework can judge if the inherited widget has changed.

Reply:
widgets should never have `==`s